<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>Dr. Kliment Olechnovič - Curriculum Vitae</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    /* The extra [class] is a hack that increases specificity enough to
       override a similar rule in reveal.js */
    ul.task-list[class]{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      font-size: inherit;
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
    .display.math{display: block; text-align: center; margin: 0.5rem auto;}
  </style>
  <link rel="stylesheet" href="pandoc.css" />
</head>
<body>
<h1 id="dr.-kliment-olechnovič">Dr. Kliment Olechnovič</h1>
<p><img style="width:200px;float:right;" src="photo_alt.jpg"></p>
<p>Updated on 2024-08-31.<br />
Latest CV online: <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/index.html">https://www.kliment.lt</a>
| <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/Kliment_Olechnovic_CV.pdf">pdf</a>.</p>
<h2 id="general-information">General information</h2>
<dl>
<dt><strong>Occupation</strong></dt>
<dd>
MSCA Postdoctoral Fellow at CNRS Laboratoire Jean Kuntzmann, Grenoble,
France (from 2023-09-01 to 2025-08-31)
</dd>
<dd>
on sabbatical leave as Senior Researcher at Vilnius University Life
Sciences Center until 2025-09-01
</dd>
<dt><strong>Email</strong></dt>
<dd>
<a href="mailto:kliment.olechnovic@bti.vu.lt"
class="email">kliment.olechnovic@bti.vu.lt</a>
</dd>
</dl>
<h3 id="online-profiles">Online profiles</h3>
<dl>
<dt><strong>Google Scholar</strong></dt>
<dd>
<a href="https://scholar.google.lt/citations?user=uT_t5ewAAAAJ"
class="uri">https://scholar.google.lt/citations?user=uT_t5ewAAAAJ</a>
</dd>
<dt><strong>ORCID</strong></dt>
<dd>
<a href="https://orcid.org/0000-0003-4918-9505"
class="uri">https://orcid.org/0000-0003-4918-9505</a>
</dd>
<dt><strong>GitHub</strong></dt>
<dd>
<a href="https://github.com/kliment-olechnovic"
class="uri">https://github.com/kliment-olechnovic</a>
</dd>
</dl>
<h3 id="main-research-interests">Main research interests</h3>
<ul>
<li>Structural bioinformatics</li>
<li>Machine learning</li>
<li>Computational geometry</li>
</ul>
<h2 id="education">Education</h2>
<dl>
<dt><strong>2012–2017</strong></dt>
<dd>
Ph.D. Computer Science, Vilnius University
</dd>
<dt><strong>2010–2012</strong></dt>
<dd>
M.S. Computer Science, Vilnius University (<em>Magna Cum Laude</em>)
</dd>
<dt><strong>2005–2009</strong></dt>
<dd>
B.S. Bioinformatics, Vilnius University
</dd>
</dl>
<h2 id="work-experience">Work experience</h2>
<dl>
<dt><strong>2023–now</strong></dt>
<dd>
MSCA Postdoctoral Fellow (CNRS Laboratoire Jean Kuntzmann, Grenoble,
France)
</dd>
<dt><strong>2020–2023</strong></dt>
<dd>
Senior Researcher (Vilnius University / Life Sciences Center / Institute
of Biotechnology)
</dd>
<dt><strong>2021–2023</strong></dt>
<dd>
Visiting researcher, for 28 weeks in total (CNRS Laboratoire Jean
Kuntzmann, Grenoble, France)
</dd>
<dt><strong>2017–2020</strong></dt>
<dd>
Researcher (Vilnius University / Life Sciences Center / Institute of
Biotechnology)
</dd>
<dt><strong>2019–2020</strong></dt>
<dd>
Visiting researcher, for 9 weeks (Inria, Grenoble, France)
</dd>
<dt><strong>2013–2017</strong></dt>
<dd>
Junior Researcher (Vilnius University / Institute of Biotechnology)
</dd>
<dt><strong>2010–2013</strong></dt>
<dd>
Research Engineer (Vilnius University / Institute of Biotechnology)
</dd>
<dt><strong>2009–2010</strong></dt>
<dd>
Research Engineer (Institute of Biotechnology, Vilnius)
</dd>
<dt><strong>2007–2008</strong></dt>
<dd>
C++ software developer (4Team Corporation, Vilnius)
</dd>
</dl>
<h2 id="publications">Publications</h2>
<h3 id="research-papers">Research papers</h3>
<ol type="1">
<li><p>(preprint) <a
href="https://doi.org/10.1101/2024.02.05.577169">Voronota-LT: efficient,
flexible and solvent-aware tessellation-based analysis of atomic
interactions</a>.<br />
<strong>Olechnovič K</strong>, Grudinin S.<br />
Preprint in <em>bioRxiv</em>. 2024.<br />
<a
href="https://doi.org/10.1101/2024.02.05.577169">doi:10.1101/2024.02.05.577169</a>.</p></li>
<li><p><a href="https://doi.org/10.1093/nar/gkae278">PPI3D: a web server
for searching, analyzing and modeling protein-protein, protein-peptide
and protein-nucleic acid interactions</a>.<br />
Dapkūnas J, Timinskas A, <strong>Olechnovič K</strong>, Tomkuvienė M,
Venclovas Č.<br />
<em>Nucleic Acids Res</em>. 2024 Apr.<br />
<a
href="https://doi.org/10.1093/nar/gkae278">doi:10.1093/nar/gkae278</a>.<br />
<a
href="https://pubmed.ncbi.nlm.nih.gov/38619046/">PMID:38619046</a>.</p></li>
<li><p><a
href="https://doi.org/10.1093/bioinformatics/btae157">TemStaPro: protein
thermostability prediction using sequence representations from protein
language models</a>.<br />
Pudžiuvelytė I, <strong>Olechnovič K</strong>, Godliauskaite E, Sermokas
K, Urbaitis T, Gasiunas G, Kazlauskas D.<br />
<em>Bioinformatics</em>. 2024 Apr 4.<br />
<a
href="https://doi.org/10.1093/bioinformatics/btae157">doi:10.1093/bioinformatics/btae157</a>.<br />
<a
href="https://pubmed.ncbi.nlm.nih.gov/38507682/">PMID:38507682</a>.</p></li>
<li><p><a href="https://doi.org/10.1002/prot.26554">VoroIF-GNN: Voronoi
tessellation-derived protein-protein interface assessment using a graph
neural network</a>.<br />
<strong>Olechnovič K</strong>, Venclovas Č.<br />
<em>Proteins</em>. 2023 Jul 21.<br />
<a
href="https://doi.org/10.1002/prot.26554">doi:10.1002/prot.26554</a>.<br />
<a
href="https://pubmed.ncbi.nlm.nih.gov/37482904/">PMID:37482904</a>.</p></li>
<li><p><a href="https://doi.org/10.1002/prot.26569">Prediction of
protein assemblies by structure sampling followed by interface-focused
scoring</a>.<br />
<strong>Olechnovič K</strong>, Valančauskas L, Dapkūnas J, Venclovas
Č.<br />
<em>Proteins</em>. 2023 Aug 14.<br />
<a
href="https://doi.org/10.1002/prot.26569">doi:10.1002/prot.26569</a>.<br />
<a
href="https://pubmed.ncbi.nlm.nih.gov/37578163/">PMID:37578163</a>.</p></li>
<li><p><a href="https://doi.org/10.1002/prot.26609">Impact of AlphaFold
on Structure Prediction of Protein Complexes: The CASP15-CAPRI
Experiment</a>.<br />
Lensink M, Brysbaert G, Raouraoua N, Bates P, Giulini M, Vargas Honorato
R, van Noort C, Teixeira J, MJJ Bonvin A, Kong R, Shi H, Lu X, Chang S,
Liu J, Guo Z, Chen X, Morehead A, Roy R, Wu T, Giri N, Quadir F, Chen C,
Cheng J, Del Carpio C, Ichiishi E, Fernández-Recio J, Harmalkar A, Chu
L, Canner S, Smanta R, Gray J, Li H, Lin P, He J, Tao H, Huang S, Roel
J, Jimenez-Garcia B, Christoffer C, Jain A, Kagaya Y, Kannan H, Nakamura
T, Terashi G, Verburgt J, Zhang Y, Zhang Z, Fujuta H, Sekijima M, Kihara
D, Khan O, Kotelnikov S, Ghani U, Padhorny D, Beglov D, Vajda S, Kozakov
D, Negi S, Ricciardelli T, Barradas-Bautista D, Cao Z, Chawla M, Cavallo
L, Oliva R, Yin R, Cheung M, Guest J, Lee J, Pierce B, Shor B, Cohen T,
Halfon M, Schneidman-Duhovny D, Zhu S, Yin R, Sun Y, Shen Y,
Maszota-Zieleniak M, Bojarski K, Lubecka E, Marcisz M, Danielsson A,
Dziadek L, Gaardlos M, Giełdoń A, Liwo J, Samsonov S, Slusarz R, Zieba
K, Sieradzan A, Czaplewski C, Kobayashi S, Miyakawa Y, Kiyota Y,
Takeda-Shitaka M, <strong>Olechnovič K</strong>, Valančauskas L,
Dapkūnas J, Venclovas C, Wallner B, Yang L, Hou C, He X, Guo S, Jiang S,
Ma X, Duan R, Qiu L, Xu X, Zou X, Velankar S, Wodak S.<br />
<em>Proteins</em>. 2023 Oct 31.<br />
<a
href="https://doi.org/10.1002/prot.26609">doi:10.1002/prot.26609</a>.<br />
<a
href="https://pubmed.ncbi.nlm.nih.gov/37905971/">PMID:37905971</a>.</p></li>
<li><p><a
href="https://doi.org/10.22541/au.167569565.51141128/v1">Discriminating
Physiological from Non-Physiological Interfaces in Structures of Protein
Complexes: A Community-Wide Study</a>.<br />
Schweke H, Xu Q, Tauriello G, Pantolini L, Schwede T, Cazals F,
Lhéritier A, Fernandez-Recio J, Rodríguez-Lumbreras LÁ, Schueler-Furman
O, Varga JK, Jiménez-García B, Réau MF, Bonvin A, Savojardo C, Martelli
P-L, Casadio R, Tubiana J, Wolfson H, Oliva R, Barradas-Bautista D,
Ricciardelli T, Cavallo L, Venclovas Č, <strong>Olechnovič K</strong>,
Guerois R, Andreani J, Martin J, Wang X, Kihara D, Marchand A, Correia
B, Zou X, Dey S, Dunbrack R, Levy E, Wodak S.<br />
<em>Proteomics</em>. 2023 Jun 27.<br />
<a
href="https://doi.org/10.1002/pmic.202200323">doi:10.1002/pmic.202200323</a>.<br />
<a
href="https://pubmed.ncbi.nlm.nih.gov/37365936/">PMID:37365936</a>.</p></li>
<li><p><a href="https://doi.org/10.1002/prot.26231">Modeling SARS-CoV2
proteins in the CASP-commons experiment</a>.<br />
Kryshtafovych A, Moult A, Billings WM, Della Corte D, Fidelis K, Kwon S,
<strong>Olechnovič K</strong>, Seok C, Venclovas Č, Won J, et al.<br />
<em>Proteins</em>. 2021 Aug 30.<br />
<a
href="https://doi.org/10.1002/prot.26231">doi:10.1002/prot.26231</a>.<br />
<a
href="https://pubmed.ncbi.nlm.nih.gov/34462960/">PMID:34462960</a>.</p></li>
<li><p><a href="https://doi.org/10.1002/prot.26222">Prediction of
protein assemblies, the next frontier: The CASP14-CAPRI
experiment</a>.<br />
Lensink MF, Brysbaert G, Mauri T, Nadzirin N, Velankar S, Chaleil RAG,
Clarence T, Bates PA, Kong R, Liu B, Yang G, Liu M, Shi H, Lu X, Chang
S, Roy RS, Quadir F, Liu J, Cheng J, Antoniak A, Czaplewski C, GiełdoŃ
A, Kogut M, Lipska AG, Liwo A, Lubecka EA, Maszota-Zieleniak M,
Sieradzan AK, Ślusarz R, Wesołowski PA, ZiĘba K, Del Carpio Muñoz CA,
Ichiishi E, Harmalkar A, Gray JJ, Bonvin AMJJ, Ambrosetti F, Honorato
RV, Jandova Z, Jiménez-García B, Koukos PI, Van Keulen S, Van Noort CW,
Réau M, Roel-Touris J, Kotelnikov S, Padhorny D, Porter KA, Alekseenko
A, Ignatov M, Desta I, Ashizawa R, Sun Z, Ghani U, Hashemi N, Vajda S,
Kozakov D, Rosell M, Rodríguez-Lumbreras LA, Fernandez-Recio J,
Karczynska A, Grudinin S, Yan Y, Li H, Lin P, Huang SY, Christoffer C,
Terashi G, Verburgt J, Sarkar D, Aderinwale T, Wang X, Kihara D,
Nakamura T, Hanazono Y, Gowthaman R, Guest JD, Yin R, Taherzadeh G,
Pierce BG, Barradas-Bautista D, Cao Z, Cavallo L, Oliva R, Sun Y, Zhu S,
Shen Y, Park T, Woo H, Yang J, Kwon S, Won J, Seok C, Kiyota Y,
Kobayashi S, Harada Y, Takeda-Shitaka M, Kundrotas PJ, Singh A, Vakser
IA, Dapkūnas J, <strong>Olechnovič K</strong>, Venclovas Č, Duan R, Qiu
L, Zhang S, Zou X, Wodak SJ.<br />
<em>Proteins</em>. 2021 Aug 28.<br />
<a
href="https://doi.org/10.1002/prot.26222">doi:10.1002/prot.26222</a>.<br />
<a
href="https://pubmed.ncbi.nlm.nih.gov/34453465/">PMID:34453465</a>.</p></li>
<li><p><a href="https://doi.org/10.1002/prot.26167">Modeling of protein
complexes in CASP14 with emphasis on the interaction interface
prediction</a>.<br />
Dapkūnas J, <strong>Olechnovič K</strong>, Venclovas Č.<br />
<em>Proteins</em>. 2021 Jun 27.<br />
<a
href="https://doi.org/10.1002/prot.26167">doi:10.1002/prot.26167</a>.<br />
<a
href="https://pubmed.ncbi.nlm.nih.gov/34176161/">PMID:34176161</a>.</p></li>
<li><p><a
href="https://doi.org/10.1093/bioinformatics/btab448">VoroContacts: a
tool for the analysis of interatomic contacts in macromolecular
structures</a>.<br />
<strong>Olechnovič K</strong>, Venclovas Č.<br />
<em>Bioinformatics</em>. 2021 Jun 16.<br />
<a
href="https://academic.oup.com/bioinformatics/advance-article-pdf/doi/10.1093/bioinformatics/btab448/38660285/btab448.pdf?guestAccessKey=ea40f0bc-eb2b-44c7-bc9b-6ffc53fac8f4">pdf</a>.<br />
<a
href="https://doi.org/10.1093/bioinformatics/btab448">doi:10.1093/bioinformatics/btab448</a>.<br />
<a
href="https://pubmed.ncbi.nlm.nih.gov/34132767">PMID:34132767</a>.</p></li>
<li><p><a href="https://doi.org/10.1093/bioinformatics/btab118">VoroCNN:
Deep convolutional neural network built on 3D Voronoi tessellation of
protein structures</a>.<br />
Igashov I, <strong>Olechnovič K</strong>, Kadukova M, Venclovas Č,
Grudinin S.<br />
<em>Bioinformatics</em>. 2021 Feb 23.<br />
<a
href="https://doi.org/10.1093/bioinformatics/btab118">doi:10.1093/bioinformatics/btab118</a>.<br />
<a
href="https://pubmed.ncbi.nlm.nih.gov/33620450">PMID:33620450</a>.</p></li>
<li><p><a href="https://doi.org/10.1002/prot.25845">Template-based
modeling of diverse protein interactions in CAPRI rounds
38-45</a>.<br />
Dapkūnas J, Kairys V, <strong>Olechnovič K</strong>, Venclovas Č.<br />
<em>Proteins</em>. 2020 Aug;88(8):939-947.<br />
<a
href="https://www.bti.vu.lt/dokumentai/skyriu_medziaga/bi/publications/2020_CAPRI_38-45_interaction_modeling_Proteins.pdf">pdf</a>.<br />
<a
href="https://doi.org/10.1002/prot.25845">doi:10.1002/prot.25845</a>.<br />
<a
href="https://www.ncbi.nlm.nih.gov/pubmed/31697420">PMID:31697420</a>.</p></li>
<li><p><a href="https://doi.org/10.1002/prot.25838">Blind prediction of
homo- and hetero-protein complexes: The CASP13-CAPRI
experiment</a>.<br />
Lensink MF, Brysbaert G, Nadzirin N, Velankar S, Chaleil RAG, Gerguri T,
Bates PA, Laine E, Carbone A, Grudinin S, Kong R, Liu RR, Xu XM, Shi H,
Chang S, Eisenstein M, Karczynska A, Czaplewski C, Lubecka E, Lipska A,
Krupa P, Mozolewska M, Golon Ł, Samsonov S, Liwo A, Crivelli S, Pagès G,
Karasikov M, Kadukova M, Yan Y, Huang SY, Rosell M, Rodríguez-Lumbreras
LA, Romero-Durana M, Díaz-Bueno L, Fernandez-Recio J, Christoffer C,
Terashi G, Shin WH, Aderinwale T, Maddhuri Venkata Subraman SR, Kihara
D, Kozakov D, Vajda S, Porter K, Padhorny D, Desta I, Beglov D, Ignatov
M, Kotelnikov S, Moal IH, Ritchie DW, Chauvot de Beauchêne I, Maigret B,
Devignes MD, Ruiz Echartea ME, Barradas-Bautista D, Cao Z, Cavallo L,
Oliva R, Cao Y, Shen Y, Baek M, Park T, Woo H, Seok C, Braitbard M,
Bitton L, Scheidman-Duhovny D, Dapkūnas J, <strong>Olechnovič
K</strong>, Venclovas Č, Kundrotas PJ, Belkin S, Chakravarty D, Badal
VD, Vakser IA, Vreven T, Vangaveti S, Borrman T, Weng Z, Guest JD,
Gowthaman R, Pierce BG, Xu X, Duan R, Qiu L, Hou J, Ryan Merideth B, Ma
Z, Cheng J, Zou X, Koukos PI, Roel-Touris J, Ambrosetti F, Geng C,
Schaarschmidt J, Trellet ME, Melquiond ASJ, Xue L, Jiménez-García B, van
Noort CW, Honorato RV, Bonvin AMJJ, Wodak SJ.<br />
<em>Proteins</em>. 2019 Dec;87(12):1200-1221.<br />
<a
href="https://doi.org/10.1002/prot.25838">doi:10.1002/prot.25838</a>.<br />
<a
href="https://www.ncbi.nlm.nih.gov/pubmed/31612567">PMID:31612567</a>.</p></li>
<li><p><a href="https://doi.org/10.1002/prot.25774">Structural modeling
of protein complexes: Current capabilities and challenges</a>.<br />
Dapkūnas J, <strong>Olechnovič K</strong>, Venclovas Č.<br />
<em>Proteins</em>. 2019 Dec;87(12):1222-1232.<br />
<a
href="https://doi.org/10.1002/prot.25774">doi:10.1002/prot.25774</a>.<br />
<a
href="https://www.ncbi.nlm.nih.gov/pubmed/31294859">PMID:31294859</a>.</p></li>
<li><p><a href="https://doi.org/10.1002/prot.25767">Estimation of model
accuracy in CASP13</a>.<br />
Cheng J, Choe MH, Elofsson A, Han KS, Hou J, Maghrabi AHA, McGuffin LJ,
Menéndez-Hurtado D, <strong>Olechnovič K</strong>, Schwede T, Studer G,
Uziela K, Venclovas Č, Wallner B.<br />
<em>Proteins</em>. 2019 Dec;87(12):1361-1377.<br />
<a
href="https://doi.org/10.1002/prot.25767">doi:10.1002/prot.25767</a>.<br />
<a
href="https://www.ncbi.nlm.nih.gov/pubmed/31265154">PMID:31265154</a>.</p></li>
<li><p><a href="https://doi.org/10.1093/nar/gkz367">VoroMQA web server
for assessing three-dimensional structures of proteins and protein
complexes</a>.<br />
<strong>Olechnovič K</strong>, Venclovas Č.<br />
<em>Nucleic Acids Res</em>. 2019 Jul 2;47(W1):W437-W442.<br />
<a
href="https://doi.org/10.1093/nar/gkz367">doi:10.1093/nar/gkz367</a>.<br />
<a
href="https://www.ncbi.nlm.nih.gov/pubmed/31073605">PMID:31073605</a>.</p></li>
<li><p><a
href="https://doi.org/10.1093/bioinformatics/bty760">Comparative
analysis of methods for evaluation of protein models against native
structures</a>.<br />
<strong>Olechnovič K</strong>, Monastyrskyy B, Kryshtafovych A,
Venclovas Č.<br />
<em>Bioinformatics</em>. 2019 Mar 15;35(6):937-944.<br />
<a
href="https://doi.org/10.1093/bioinformatics/bty760">doi:10.1093/bioinformatics/bty760</a>.<br />
<a
href="https://www.ncbi.nlm.nih.gov/pubmed/30169622">PMID:30169622</a>.</p></li>
<li><p><a href="https://doi.org/10.1002/prot.25378">Modeling of protein
complexes in CAPRI Round 37 using template-based approach combined with
model selection</a>.<br />
Dapkūnas J, <strong>Olechnovič K</strong>, Venclovas Č.<br />
<em>Proteins</em>. 2018 Mar;86 Suppl 1:292-301.<br />
<a
href="https://doi.org/10.1002/prot.25378">doi:10.1002/prot.25378</a>.<br />
<a
href="https://www.ncbi.nlm.nih.gov/pubmed/28905467">PMID:28905467</a>.</p></li>
<li><p><a href="https://doi.org/10.1002/prot.25278">VoroMQA: Assessment
of protein structure quality using interatomic contact areas</a>.<br />
<strong>Olechnovič K</strong>, Venclovas Č.<br />
<em>Proteins</em>. 2017 Jun;85(6):1131-1145.<br />
<a
href="https://doi.org/10.1002/prot.25278">doi:10.1002/prot.25278</a>.<br />
<a
href="https://www.ncbi.nlm.nih.gov/pubmed/28263393">PMID:28263393</a>.</p></li>
<li><p><a href="https://doi.org/10.1093/bioinformatics/btw756">The PPI3D
web server for searching, analyzing and modeling protein-protein
interactions in the context of 3D structures</a>.<br />
Dapkūnas J, Timinskas A, <strong>Olechnovič K</strong>, Margelevičius M,
Diciunas R, Venclovas Č.<br />
<em>Bioinformatics</em>. 2017 Mar 15;33(6):935-937.<br />
<a
href="https://doi.org/10.1093/bioinformatics/btw756">doi:10.1093/bioinformatics/btw756</a>.<br />
<a
href="https://www.ncbi.nlm.nih.gov/pubmed/28011769">PMID:28011769</a>.</p></li>
<li><p><a href="https://doi.org/10.1093/nar/gku294">The CAD-score web
server: contact area-based comparison of structures and interfaces of
proteins, nucleic acids and their complexes</a>.<br />
<strong>Olechnovič K</strong>, Venclovas Č.<br />
<em>Nucleic Acids Res</em>. 2014 Jul;42(Web Server issue):W259-63.<br />
<a
href="https://doi.org/10.1093/nar/gku294">doi:10.1093/nar/gku294</a>.<br />
<a
href="https://www.ncbi.nlm.nih.gov/pubmed/24838571">PMID:24838571</a>.</p></li>
<li><p><a href="https://doi.org/10.1093/nar/gku191">The use of
interatomic contact areas to quantify discrepancies between RNA 3D
models and reference structures</a>.<br />
<strong>Olechnovič K</strong>, Venclovas Č.<br />
<em>Nucleic Acids Res</em>. 2014 May;42(9):5407-15.<br />
<a
href="https://doi.org/10.1093/nar/gku191">doi:10.1093/nar/gku191</a>.<br />
<a
href="https://www.ncbi.nlm.nih.gov/pubmed/24623815">PMID:24623815</a>.</p></li>
<li><p><a href="https://doi.org/10.1002/jcc.23538">Voronota: A fast and
reliable tool for computing the vertices of the Voronoi diagram of
atomic balls</a>.<br />
<strong>Olechnovič K</strong>, Venclovas Č.<br />
<em>J Comput Chem</em>. 2014 Mar 30;35(8):672-81.<br />
<a
href="https://doi.org/10.1002/jcc.23538">doi:10.1002/jcc.23538</a>.<br />
<a
href="https://www.ncbi.nlm.nih.gov/pubmed/24523197">PMID:24523197</a>.</p></li>
<li><p><a href="https://doi.org/10.1002/prot.24172">CAD-score: a new
contact area difference-based function for evaluation of protein
structural models</a>.<br />
<strong>Olechnovič K</strong>, Kulberkytė E, Venclovas Č.<br />
<em>Proteins</em>. 2013 Jan;81(1):149-62.<br />
<a
href="https://doi.org/10.1002/prot.24172">doi:10.1002/prot.24172</a>.<br />
<a
href="https://www.ncbi.nlm.nih.gov/pubmed/22933340">PMID:22933340</a>.</p></li>
<li><p><a href="https://doi.org/10.1093/bioinformatics/btq720">Voroprot:
an interactive tool for the analysis and visualization of complex
geometric features of protein structure</a>.<br />
<strong>Olechnovič K</strong>, Margelevičius M, Venclovas Č.<br />
<em>Bioinformatics</em>. 2011 Mar 1;27(5):723-4.<br />
<a
href="https://doi.org/10.1093/bioinformatics/btq720">doi:10.1093/bioinformatics/btq720</a>.<br />
<a
href="https://www.ncbi.nlm.nih.gov/pubmed/21186248">PMID:21186248</a>.</p></li>
</ol>
<h3 id="book-chapters">Book chapters</h3>
<ul>
<li><p><a href="https://doi.org/10.1007/978-1-0716-0270-6_6">Contact
Area-Based Structural Analysis of Proteins and Their Complexes Using
CAD-Score</a>.<br />
<strong>Olechnovič K</strong>, Venclovas Č.<br />
In: <em>Zoltán Gáspári (eds) Structural Bioinformatics: Methods and
Protocols, Methods in Molecular Biology, vol. 2112</em>. Springer.
2020.</p></li>
<li><p><a href="https://doi.org/10.1007/978-3-030-12780-0_15">In Silico
Modeling of Inhibitor Binding to Carbonic Anhydrases</a>.<br />
Kairys V, <strong>Olechnovič K</strong>, Raškevičius V, Matulis D.<br />
In: <em>Matulis D. (eds) Carbonic Anhydrase as Drug Target</em>.
Springer, Cham. 2019.</p></li>
</ul>
<h3 id="doctoral-dissertation">Doctoral dissertation</h3>
<ul>
<li><p><a
href="https://epublications.vu.lt/object/elaba:22955146/">Methods for
the analysis and assessment of the three-dimensional structures of
proteins and nucleic acids: development and applications</a>.<br />
<strong>Olechnovič K</strong>.<br />
Doctoral dissertation, Vilnius University, 2017.</p></li>
<li><p><a
href="https://epublications.vu.lt/object/elaba:22955391/">Baltymų ir
nukleorūgščių erdvinių struktūrų analizės ir vertinimo metodai: kūrimas
ir taikymas</a>.<br />
<strong>Olechnovič K</strong>.<br />
Doctoral dissertation summary in Lithuanian, Vilnius University,
2017.</p></li>
</ul>
<h3 id="other-publications">Other publications</h3>
<ul>
<li><p><a
href="https://issuu.com/vu_lt/docs/spectrum26.2017/6">Kompiuteriai
padeda pažinti sudėtingą baltymų pasaulį</a>.<br />
Dapkūnas J, <strong>Olechnovič K</strong>.<br />
Popular science article in <em>SPECTRUM</em>. 2017 1(26), ISSN
1822-0147.</p></li>
<li><p><a href="https://doi.org/10.1002/prot.25129">Journal cover image
based on the article <em>“VoroMQA: Assessment of protein structure
quality using interatomic contact areas”</em></a>.<br />
<strong>Olechnovič K</strong>, Venclovas Č.<br />
Cover Image for <em>Proteins</em>. 2019 Volume 85, Issue 6.<br />
<a
href="https://doi.org/10.1002/prot.25129">doi:10.1002/prot.25129</a>.</p></li>
</ul>
<h2 id="presentations-and-posters">Presentations and posters</h2>
<h3 id="oral-presentations-at-international-conferences">Oral
presentations at international conferences</h3>
<ul>
<li>Joint ICB Workshop and 8th CAPRI Assessment Meeting, France,
Grenoble (2024)</li>
<li>CASP15 meeting, Turkey, Antalya (2022)</li>
<li>AI at CIRM, France, Marseille (2021)</li>
<li>CASP14 meeting, Virtual (2020)</li>
<li>COINS, Lithuania, Vilnius (2019), <em>keynote presentation</em></li>
<li>CASP13 meeting, Mexico, Riviera Maya (2018)</li>
<li>VitaScientia, Lithuania, Vilnius (2018)</li>
<li>CASP12 meeting, Italy, Gaeta (2016)</li>
<li>CASP10 meeting, Italy, Gaeta (2012)</li>
</ul>
<h3 id="poster-presentations-at-international-conferences">Poster
presentations at international conferences</h3>
<ul>
<li>CASP15 meeting, Antalya, Turkey (2022), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/CASP_2022.png">poster</a></li>
<li>AI at CIRM, France, Marseille (2021), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/AIatCIRM_2021.png">poster</a></li>
<li>PDB50, Virtual (2021), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/PDB50_2021.png">poster</a></li>
<li>CASP14 meeting, Virtual (2020), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/CASP_2020.png">poster</a></li>
<li>ISMB, Switzerland, Basel (2019), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/ISMB_2019.png">poster</a></li>
<li>CASP13 meeting, Mexico, Riviera Maya (2018), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/CASP_2018.png">poster</a></li>
<li>CASP12 meeting, Italy, Gaeta (2016), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/CASP_2016.png">poster</a></li>
<li>ECCB, Netherlands, Hague (2016), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/ECCB_2016.png">poster</a></li>
<li>CASP11 meeting, Mexico, Riviera Maya (2014), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/CASP_2014.png">poster</a></li>
<li>ECCB, France, Strasbourg (2014), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/ECCB_2014.png">poster</a></li>
<li>ISMB, Germany, Berlin (2013), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/ISMB_2013.png">poster</a></li>
<li>SocBiN, Poland, Torun (2013), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/SocBiN_2013.png">poster</a></li>
<li>CASP10 meeting, Italy, Gaeta (2012), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/CASP_2012.png">poster</a></li>
<li>ECCB, Switzerland, Basel (2012), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/ECCB_2012.png">poster</a></li>
<li>SAGA, Lithuania, Vilnius (2011), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/SAGA_2011.png">poster</a></li>
<li>ISMB, Austria, Vienna (2011), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/ISMB_2011.png">poster</a></li>
<li>ECCB, Belgium, Ghent (2010), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/ECCB_2010.png">poster</a></li>
<li>VizBi, Germany, Heidelberg (2010), <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/VizBi_2010.png">poster</a></li>
</ul>
<p>Gallery of posters: <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/index.html">www.kliment.lt/posters</a></p>
<p><a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/index.html"><img src="posters/montage.png"></a></p>
<h2 id="main-published-software">Main published software</h2>
<ul>
<li><p>Voronota: a standalone software package of various tools for
analyzing three-dimensional structures of biological macromolecules
using the Voronoi diagram of atomic balls (includes the latest versions
of CAD-score and VoroMQA methods).<br />
<a
href="https://kliment-olechnovic.github.io/voronota/">https://kliment-olechnovic.github.io/voronota</a></p></li>
<li><p>Voronota-JS: an expansion of the core Voronota software.
Voronota-JS provides a way to write JavaScript scripts for the
comprehensive analysis of macromolecular structures, including the
Voronoi tesselation-based analysis. <a
href="https://kliment-olechnovic.github.io/voronota/expansion_js/">https://kliment-olechnovic.github.io/voronota/expansion_js/</a></p></li>
<li><p>Voronota-LT: an expansion of the core Voronota software.
Voronota-LT (pronounced ‘voronota lite’) is an alternative,
significantly faster version of Voronota for constructing
tessellation-derived atomic contact areas and volumes. <a
href="https://kliment-olechnovic.github.io/voronota/expansion_lt/">https://kliment-olechnovic.github.io/voronota/expansion_lt/</a></p></li>
<li><p>Voronota-GL: a visual tool for the comprehensive interactive
analysis of macromolecular structures, including the Voronoi
tesselation-based analysis. <a
href="https://kliment-olechnovic.github.io/voronota/expansion_gl/">https://kliment-olechnovic.github.io/voronota/expansion_gl/</a></p></li>
<li><p>FTDMP: a software system for running docking experiments and
scoring/ranking multimeric models (includes VoroIF-jury and VoroIF-GNN
methods). <a
href="https://kliment-olechnovic.github.io/ftdmp/">https://kliment-olechnovic.github.io/ftdmp/</a></p></li>
<li><p>VoroMQA web server for the assessment of protein structure
quality using interatomic contact areas.<br />
<a
href="https://bioinformatics.lt/wtsam/voromqa">https://bioinformatics.lt/wtsam/voromqa</a></p></li>
<li><p>VoroContacts web server for the computation and interactive
querying of Voronoi tessellation-derived contacts.<br />
<a
href="https://bioinformatics.lt/wtsam/vorocontacts/submit">https://bioinformatics.lt/wtsam/vorocontacts</a></p></li>
<li><p>CAD-score web server for contact area-based comparison of
structures and interfaces of proteins, nucleic acids and their
complexes.<br />
<a
href="https://bioinformatics.lt/cad-score">https://bioinformatics.lt/cad-score/</a></p></li>
<li><p>Voroprot: an interactive tool for exploring some
tesselation-derived features of protein structures (no longer
maintained).<br />
<a
href="https://bioinformatics.lt/software/voroprot">https://bioinformatics.lt/software/voroprot</a></p></li>
</ul>
<h2 id="achievements-and-awards">Achievements and awards</h2>
<h3 id="achievements-in-casp-and-capri-experiments">Achievements in CASP
and CAPRI experiments</h3>
<p>CASP (Critical Assessment of Techniques for Protein Structure
Prediction) and CAPRI (Critical Assessment of PRedicted Interactions)
are world-wide experiments focused on the blind testing of methods for
protein structural bioinformatics.</p>
<dl>
<dt><strong>2022</strong></dt>
<dd>
Contributed to one of the top performances in modeling structures of
protein complexes in CASP15 and CASP15-CAPRI experiments (ranked 2nd in
CASP, ranked 1st in CAPRI). Group “Venclovas”, members: Olechnovič K,
Valančauskas L, Dapkūnas J, Venclovas Č.
</dd>
<dt><strong>2022</strong></dt>
<dd>
One of the top performances in EMA (estimation of model accuracy) in
CASP15 experiment (and ranked 1st in CASP15-CAPRI scoring experiment).
Groups “Venclovas” and “VoroIF”.
</dd>
<dt><strong>2020</strong></dt>
<dd>
Contributed to one of the top performances in modeling structures of
protein complexes in CASP14 and CASP14-CAPRI experiments (ranked 2nd in
CASP, ranked 1st in CAPRI jointly with two other groups). Group
“Venclovas”, members: Olechnovič K, Dapkūnas J, Venclovas Č.
</dd>
<dt><strong>2019</strong></dt>
<dd>
Contributed to one of the top performances (ranked 3rd) in modeling
structures of protein complexes in the CAPRI experiment rounds 38–45.
Group “Venclovas”, members: Dapkūnas J, Kairys V, Olechnovič K,
Venclovas Č.
</dd>
<dt><strong>2018</strong></dt>
<dd>
Contributed to the best results (ranked 1st) in modeling structures of
protein complexes in CASP13 and CASP13-CAPRI experiments. Group
“Venclovas”, members: Dapkūnas J, Olechnovič K, Venclovas Č.
</dd>
<dt><strong>2018</strong></dt>
<dd>
One of the top performances in EMA (estimation of model accuracy) in
CASP13 experiment (ranked 1st in prediction of unreliable regions).
Groups “VoroMQA-A” and “VoroMQA-B”.
</dd>
<dt><strong>2016</strong></dt>
<dd>
Contributed to the best results (ranked 1st) in modeling structures of
protein complexes in CASP12-CAPRI experiment. Group “Venclovas”,
members: Dapkūnas J, Olechnovič K, Venclovas Č.
</dd>
<dt><strong>2016</strong></dt>
<dd>
One of the top performances in protein structure prediction in CASP12
experiment. Group “VoroMQA-select”.
</dd>
</dl>
<h3 id="national-awards">National awards</h3>
<dl>
<dt><strong>2019</strong></dt>
<dd>
Lithuanian Academy of Sciences scholarship for young scientists
</dd>
<dt><strong>2018</strong></dt>
<dd>
Laureate of the “Best doctoral dissertation of 2017 in Lithuania”
contest
</dd>
<dt><strong>2015</strong></dt>
<dd>
Lithuanian Academy of Sciences award for the best works by young
researchers in 2014
</dd>
<dt><strong>2013–2014</strong></dt>
<dd>
The Research Council of Lithuania scholarship for PhD students actively
conducting scientific research
</dd>
<dt><strong>2013</strong></dt>
<dd>
INFOBALT incentive scholarship for young scientists
</dd>
</dl>
<h3 id="conference-awards">Conference awards</h3>
<dl>
<dt><strong>2019</strong></dt>
<dd>
ISCB Art in Science Award Winner at “ISMB/ECCB 2019”. Work title: <a
href="https://kliment-olechnovic.github.io/curriculum-vitae/posters/disassembled_tessellation.jpg">“Disassembled
tessellation”</a>.
</dd>
<dt><strong>2016</strong></dt>
<dd>
Poster selected for an oral presentation at “12th Community Wide
Experiment on the Critical Assessment of Techniques for Protein
Structure Prediction (CASP12 meeting)”. Poster title: <em>VoroMQA:
assessment of protein structure quality using interatomic contact areas
derived from the Voronoi tessellation of atomic balls</em>.
</dd>
<dt><strong>2013</strong></dt>
<dd>
Best poster award at “Society for Bioinformatics in Northern European
countries (SocBiN)”. Poster title: <em>The use of interatomic contact
areas for the assessment of RNA 3D structural models</em>.
</dd>
<dt><strong>2012</strong></dt>
<dd>
Poster selected for an oral presentation at “EMBO Conference on Critical
Assessment of Protein Structure Prediction (CASP10 meeting)”. Poster
title: <em>CAD-score: a new method for the evaluation of protein
structural models</em>.
</dd>
</dl>
<h3 id="other-achievements">Other achievements</h3>
<dl>
<dt><strong>2018</strong></dt>
<dd>
Judo champion of Lithuania, judo black belt.
</dd>
<dt><strong>2018</strong></dt>
<dd>
Sambo champion of Lithuania.
</dd>
</dl>
</body>
</html>
