<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <title>Structural Bioinformatics Workshow on 2024.03.26-27</title>
</head>
<body>

<h1>Structural Bioinformatics Workshow on 2024.03.26-27</h1>

<h2>Example protein</h2>

<p>
For an educational example, we will "cheat" a little bit.
Let's take a small protein that already has a structure in PDB - <b>5ZKT</b> (<a href="https://www.rcsb.org/structure/5zkt">https://www.rcsb.org/structure/5zkt</a>).
</p>

<p>
But let's not look at the structure right away, but start with the full sequence - UniProt entry <b>Q0D8K4</b> (<a href="https://www.uniprot.org/uniprotkb/Q0D8K4/entry">https://www.uniprot.org/uniprotkb/Q0D8K4/entry</a>).
</p>

<h2>Getting structural models</h2>

<h3>Finding available structural predictions</h3>

<p>
UniProt entry <b>Q0D8K4</b> has an already generated prediction in the AlphaFold Protein Structure Database - <a href="https://alphafold.ebi.ac.uk/entry/Q0D8K4">https://alphafold.ebi.ac.uk/entry/Q0D8K4</a>
</p>

<p>
But we see that this prediction is not very useful - it is too long and only monomeric.
</p>

<h3>Generating structural predictions</h3>

<p>
Let's focus on the annotated functional domain (the TCP domain), here is its sequence in FASTA format:
</p>

<pre>
>Q0D8K4_TCP
GKDRHSKVYTAKGIRDRRVRLSVSTAIQFYDLQDRLGYDQPSKAIEWLIKAAAAAIDKL
</pre>

<p>
Let's go to <a href="https://github.com/sokrypton/ColabFold">https://github.com/sokrypton/ColabFold</a>.
Let us use the "AlphaFold2_mmseqs2" notebook that constructs multiple sequence alignments very rapidly and passes them to AlphaFold-Multimer.
</p>

<p>
Let's model a monomer first. If everything goes OK, results similar to the following are to be expected (notice that the pTM are not very high):
</p>

<pre>
2024-03-25 13:14:54,406 rank_001_alphafold2_ptm_model_5_seed_000 pLDDT=93.8 pTM=0.703
2024-03-25 13:14:54,407 rank_002_alphafold2_ptm_model_2_seed_000 pLDDT=93.5 pTM=0.677
2024-03-25 13:14:54,407 rank_003_alphafold2_ptm_model_4_seed_000 pLDDT=92.4 pTM=0.659
2024-03-25 13:14:54,408 rank_004_alphafold2_ptm_model_3_seed_000 pLDDT=91.8 pTM=0.659
2024-03-25 13:14:54,409 rank_005_alphafold2_ptm_model_1_seed_000 pLDDT=86.1 pTM=0.585
</pre>

<p>
Let's model a homo-dimer. If everything goes OK, high-scoring results similar to the following are to be expected (notice that the dimer has higher pTM scores):
</p>

<pre>
2024-03-25 13:03:19,899 rank_001_alphafold2_multimer_v3_model_3_seed_000 pLDDT=94.7 pTM=0.882 ipTM=0.879
2024-03-25 13:03:19,901 rank_002_alphafold2_multimer_v3_model_1_seed_000 pLDDT=94.8 pTM=0.882 ipTM=0.878
2024-03-25 13:03:19,901 rank_003_alphafold2_multimer_v3_model_4_seed_000 pLDDT=94.4 pTM=0.877 ipTM=0.874
2024-03-25 13:03:19,902 rank_004_alphafold2_multimer_v3_model_5_seed_000 pLDDT=94.3 pTM=0.876 ipTM=0.872
2024-03-25 13:03:19,903 rank_005_alphafold2_multimer_v3_model_2_seed_000 pLDDT=94.2 pTM=0.874 ipTM=0.87
</pre>

<p>
Here are the ColabFold result archives in case ColabFold is not accessible during the workshop:
<a href="data/test_mono_dd6de.result.zip">monomer results</a>,
<a href="data/test_fc94d.result.zip">dimer results</a>.
</p>

<p>
Here is the ColabFold result <a href="data/unrelaxed_rank_001.pdb">first-ranked dimeric structural model</a> and <a href="data/unrelaxed_rank_001_chain_A.pdb">its first chain</a>.
</p>

<h2>Homology-based inference</h2>

<p>
<i>"Nothing in Biology Makes Sense Except in the Light of Evolution"</i> - Theodosius Dobzhansky.
</p>

<h3>Using sequence</h3>

<p>
Perhaps the most common and efficient way to quickly look for homologs of a single sequence is <a href="https://toolkit.tuebingen.mpg.de/tools/hhpred">HHpred</a>.
</p>

<p>
Let's use <a href="https://bioinformatics.lt/ppi3d/">PPI3D</a>
to learn more about the dimer by looking at complexes containing proteins of similar sequences.
</p>

<p>
Using single-sequence BLAST search we get <a href="https://bioinformatics.lt/ppi3d/site/results/XBACUPbl">https://bioinformatics.lt/ppi3d/site/results/XBACUPbl</a>.
</p>

<p>
This allows us to generate a model of a <a href="data/tbm_protein_DNA.pdb">partial protein-DNA complex</a> that can be superimposed with the AlphaFold-generated dimer to get a rough <a href="data/protein_dna_model.pdb">full protein-DNA complex model</a>.
</p>

<h3>Using structure</h3>

<p>
Let's submit the first chain of our best model to <a href="https://search.foldseek.com/search">Foldseek</a> to quickly find similar proteins.
</p>

<p>
You can submit the first chain of our best model to <a href="http://ekhidna2.biocenter.helsinki.fi/dali/">DALI</a> to find similar proteins not very quickly, but using a more reliable alignment procedure.
The resulting job link <a href="http://ekhidna2.biocenter.helsinki.fi/barcosel/tmp//kxciECtAYWF/">http://ekhidna2.biocenter.helsinki.fi/barcosel/tmp/kxciECtAYWF/</a>
</p>

<h2>Analysis of structures</h2>

<h3>Analyzing whether a structure is relatively realistic</h3>

<p>
Let's use VoroMQA to assess the realism of our models: <a href="https://bioinformatics.lt/wtsam/voromqa/job-voromqa-MSMA2X">https://bioinformatics.lt/wtsam/voromqa/job-voromqa-MSMA2X</a>.
</p>

<p>
VoroMQA can also be used to select best models out of many, especially useful whn models com from sorces different than AlphaFold: <a href="https://bioinformatics.lt/wtsam/voromqa/job-voromqa-A8PUHU">https://bioinformatics.lt/wtsam/voromqa/job-voromqa-A8PUHU</a>.
</p>

<h3>Analysis of interesting contacts</h3>

<p>
Let's use VoroContacts to analyze what are the most important interface contacts: <a href="https://bioinformatics.lt/wtsam/vorocontacts/job-vorocontacts-LPNG6A">https://bioinformatics.lt/wtsam/vorocontacts/job-vorocontacts-LPNG6A</a>.
</p>

</body>
</html>

